backend/src/main/java/gov/cdc/usds/simplereport/db/repository/AdvisoryLockManager.java

```diff
@@ -16,22 +15,6 @@ public interface AdvisoryLockManager {
   int TEST_ORDER_LOCK_SCOPE =
       895417283; // another arbitrary number - this one specific to TestOrder
 
-  /**
-   * Take the advisory lock defined by the two arguments, waiting until the lock is available and
-   * releasing it at the end of the current transaction.
-   *
-   * @param lockCategory the high-level group of locks that contains the lock we are trying to
-   *     obtain (in this case, almost always {{@link #CORE_API_LOCK_SCOPE}).
-   * @param lock the specific lock we are trying to obtain (usually a constant in the implementing
-   *     repository or the service that calls it).
-   */
-  @Query(
-      nativeQuery = true,
-      // can't tell hibernate that Types.OTHER is a "void" result in this case:
-      // just cast it to text
-      value = "select cast(pg_advisory_xact_lock(:lockCategory, :lock) as text)")
-  void waitForTransactionLock(int lockCategory, int lock);
-
```

Comment:
this was already not being used - not related to my changes but I cleaned it up anyway